blazor/mention/templates.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Templates in Blazor Mention Component | Syncfusion
+description: Checkout and learn here all about templates in Syncfusion Blazor Mention component and much more.
+platform: Blazor
+control: Mention
+documentation: ug
+---
+
+# Templates in Blazor Mention Component
+
+Mention is a templated blazor component, that allow you to customize various part of the UI using template parameters. It allows you to render custom components or content based on your own logic. The Mention has been provided with several options to customize each suggestion list items.
+
+## Template context
+
+The templates used by the Mention component are of type `RenderFragment`, which is a special type of delegate that represents a block of Razor code that can be rendered as part of a component's user interface. The templates used by the Mention component are passed parameters that can be accessed using an implicit parameter named `context`.
+
+## Item template
+
+The `ItemTemplate` property allows you to specify a custom template for each individual suggestion list item in the Mention component. The `ItemTemplate` template is passed a parameter called `context`, which contains information about the current item being rendered. You can use this `context` parameter to access the data for the current item and use it to customize the content and appearance of the list item.
+
+In the following sample, each list item is split into two columns to display relevant data using `ItemTemplate`.
+
+{% highlight razor %}
+
+{% include_relative code-snippet/item-template.razor %}
+
+{% endhighlight %}
+
+![Blazor Mention with item template](./images/blazor-mention-item-template.png)
+
+## Display template
+
+The `DisplayTemplate` property allows you to specify a template that defines how the mentioned value should be displayed in the Mention component. The `DisplayTemplate` template is passed a parameter called `context`, which contains information about the current item being rendered. You can use this `context` parameter to access the data for the current item and use it to customize the appearance of the mentioned value, such as by adding an avatar or displaying additional information about the mentioned value.
+
+In the following sample, the selected value is displayed as a combined text of both `FirstName` and `City` in the Mention element, which is separated by a hyphen.
```

Comment:
both `FirstName` and `City` 
 
 Are you using "City"?

Check the all sample explanation

Reply:
Addressed